src/script/interpreter.cpp

```diff
@@ -973,11 +978,11 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                 break;
 
                 default:
-                    return set_error(serror, SCRIPT_ERR_BAD_OPCODE);
+                    return protocol_cleanup ? true : set_error(serror, SCRIPT_ERR_BAD_OPCODE);
```

Comment:
See above comment on returning `set_success(serror)` instead of `true` as well as thoughts on a `verify_badopcodes` type policy flag.